core/src/main/java/com/opensymphony/xwork2/ognl/OgnlUtil.java

```diff
@@ -78,14 +78,45 @@
     private boolean allowStaticMethodAccess;
     private boolean disallowProxyMemberAccess;
 
+    /**
+     * Construct a new OgnlUtil instance for use with the framework
+     */
     public OgnlUtil() {
+        this(null, null);  // Instantiate default Expression and BeanInfo caches (null factories)
+    }
```

Comment:
Is this used elsewhere in the code? I would suggest mark this constructor at least `deprecated` if not to remove it.

Reply:
It is used in some of the unit test code.  Since it is a pre-existing public constructor, marking it as `deprecated` seems like the safer option.  The latest commit to the PR does that.